components/new-contribution-flow/ContributorCardWithTier.js

```diff
@@ -83,7 +84,7 @@ const ContributorCardWithTier = ({ collective, contribution, ...props }) => {
             <FormattedMessage
               id="NewContributionFlow.CollectiveAndTier"
               defaultMessage="{collective} - {tier}"
-              values={{ collective: collective.name, tier: contribution.exampleTier }}
+              values={{ collective: collective.name, tier: contribution.tier?.name || 'Donor' }}
```

Comment:
`Donor` should be internationalized, and according to https://docs.opencollective.com/help/about/terminology I think we want to show `Contributor` instead